src/js/api/order.js

```diff
@@ -2152,11 +2158,51 @@ ripe.Ripe.prototype.updateTagOrderP = function(number, identifier, type, options
     });
 };
 
+/**
+ * Deletes the tag for an order, by default, deactivates it as well.
+ *
+ * @param {Number} number The number of the order with the tag to delete.
+ * @param {Object} options An object of options to configure the request.
+ * @param {Function} callback Function with the result of the request.
+ * @returns {XMLHttpRequest} The order.

```

Comment:
This should be (see older documentation of this file)
```suggestion
 * @returns {XMLHttpRequest} Resulting information for the callback execution.

```